apps/app/vite.config.ts

```diff
@@ -76,6 +76,7 @@ export default defineConfig(({ mode }) => {
 						'/components/avatar',
 						'/components/badge',
 						'/components/button',
+						'/components/breadcrumb',
```

Comment:
`breakdcrumb` should be before `button` in alphabetic order

Reply:
I fixed it, and also reorder breadcrumb position in another files too.
Thanks